src/Toolkit/Toolkit/Internal/VisibilityConverter.cs

```diff
@@ -31,7 +31,7 @@ namespace Esri.ArcGISRuntime.Toolkit.Internal
     /// *FOR INTERNAL USE* Returns visible status for positive boolean, non-null text and opposite state for visibility value.
     /// </summary>
     [System.ComponentModel.EditorBrowsable(System.ComponentModel.EditorBrowsableState.Never)]
-    public class VisibilityConverter : IValueConverter
+    public sealed class VisibilityConverter : IValueConverter
     {
         /// <inheritdoc />
         object IValueConverter.Convert(object value, Type targetType, object parameter,
```

Comment:
I believe `value` param should be nullable here.

Hopefully the Xamarin.Forms or Maui team goes back and adds nullability annotations to their public interfaces some day...

Reply:
Good catch. `IValueConverter` is part of core .NET and was one of the few classes that didn't make it for .NET 5.